internal/server/polling/terraform.go

```diff
@@ -38,6 +62,119 @@ func (s *Server) getSource(ctx context.Context, tf *v1alpha2.Terraform) (*source
 	return obj, nil
 }
 
+func (s *Server) createTerraform(ctx context.Context, originalTF *infrav1.Terraform, originalSource *sourcev1b2.GitRepository, branch string, prID string, interval time.Duration) error {
+	tf := originalTF.DeepCopy()
+
+	tf.ResourceVersion = ""
+
+	tfName := s.createObjectName(tf.ObjectMeta.Name, branch, prID)
+
+	tf.SetName(tfName)
+
+	tf.SetLabels(s.createLabels(tf.Labels, branch, prID))
+
+	msg := fmt.Sprintf("Terraform object %s in the namespace %s", tfName, tf.ObjectMeta.Namespace)
```

Comment:
Can we group variable assignments and functions calls together? I like (and highly prefer) when the code has empty lines to break up logical sections, but not between every lines.